server/src/internalClusterTest/java/org/elasticsearch/indices/state/ReopenWhileClosingIT.java

```diff
@@ -81,11 +79,7 @@ public void testReopenDuringClose() throws Exception {
     }
 
     public void testReopenDuringCloseOnMultipleIndices() throws Exception {
-        // need two management threads here since we're blocking one below
-        List<String> dataOnlyNodes = internalCluster().startDataOnlyNodes(
-            randomIntBetween(2, 3),
-            Settings.builder().put("thread_pool.management.core", 2).put("thread_pool.management.max", 2).build()
```

Comment:
I mean the settings change.

Reply:
I added this to 7.17 when backporting thinking it would fix the issue in #92629 but these settings wouldn't even get applied so it was pointless -> my bad in the first place :)